### PR TITLE
requirements update for spacy and lightgbm

### PR DESCRIPTION
**Pull request by stephaniewhoo**
+ add spacy 3.0.6 and lightgbm 3.2.1 to the requirement

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,3 +9,5 @@ transformers>=4.6.0
 sentencepiece>=0.1.95
 nmslib>=2.1.1
 onnxruntime>=1.8.1
+spacy==3.0.6
```
Comment (lintool) [inline requirements.txt:12]: @MXueguang @ronakice @yuki617 - what do you think? should it be pegged to a version?

`>=` instead? Or just unspecified?

FWIW, my system:

```
lightgbm           3.3.2
spacy              3.2.1
```

Comment (stephaniewhoo) [reply requirements.txt:12]: ah yes, I think `>=` is a better idea
if using `>=`, should we use newer version instead of my old ones

Comment (MXueguang) [reply requirements.txt:12]: @ronakice  does it affect the 2.1.6 restriction for d2q?

Comment (MXueguang) [reply requirements.txt:12]: I see. yeah then >= 3.2.1 would be good

Comment (lintool) [reply requirements.txt:12]: So @stephaniewhoo let's do

```
lightgbm>=3.3.2
spacy>=3.2.1
```

Comment (ronakice) [reply requirements.txt]: yeah, for v1 d2q with the old stuff we can just leave it and mention in the repository clearly that we used that.

Comment (yuki617) [reply requirements.txt]: Yea, I believe spacy>=3.2.1 should be good.

**Review by lintool — APPROVED** [review]

**Comment by lintool** [comment]
🚢 it!

**Pull request merged**